multigeiger/multigeiger.ino

```diff
@@ -917,8 +948,8 @@ void sendData2TTN(int sendwhat, unsigned int hvpulses) {
   // next byte is the tube version
   ttnData[4] = tubes[TUBE_TYPE].nbr;
     // and last is software version
-  ttnData[5] = SOFTWARE_VERSION>>8;
-  ttnData[6] = SOFTWARE_VERSION&0xFF;
+  ttnData[5] = (unsigned char)lora_software_version[0];
+  ttnData[6] = (unsigned char)lora_software_version[1];
```

Comment:
how about typing lora_software_version variable as `uint8 ... []` (or unsigned char, if uint8 does not exist)?

Reply:
Could be done. Then we should cast the atoi to unsigned char.